chore(templates/website): adds seeded users to readme (#3412)
payloadcms/payload: templates/website/README.md

To seed the database with a few pages, posts, and projects you can run `yarn seed`. This template also comes with a `GET /api/seed` endpoint you can use to seed the database from the admin panel.

> On boot, a seed script is included to create 2 users:

Reviewer: ```suggestion
The seed script will also create two users for demonstration purposes only:
```

> 1. email `demo-author@payloadcms.com`, password `password`, the role `admin`.

Reviewer: ```suggestion
1. Demo Author
    - Email: `demo-author@payloadcms.com`
    - Password: `password`
    - Role: `admin`
```

> 2. email `demo-user@payloadcms.com`, password `password`, the role `user`

Reviewer: ```suggestion
2. Demo User
    - Email: `demo-user@payloadcms.com`
    - Password: `password`
    - Role: `user`
```
